rskj-core/src/main/java/org/ethereum/facade/EthereumImpl.java

```diff
@@ -93,19 +92,19 @@ public EthereumImpl(RskSystemProperties config,
 
     @Override
     public void init() {
-        if (config.listenPort() > 0) {
+        if (config.peerPort() > 0) {
             peerServiceExecutor = Executors.newSingleThreadExecutor(runnable -> {
                 Thread thread = new Thread(runnable, "Peer Server");
-                thread.setUncaughtExceptionHandler((exceptionThread, exception) -> {
-                    gLogger.error("Unable to start peer server", exception);
-                });
+                thread.setUncaughtExceptionHandler((exceptionThread, exception) ->
+                    gLogger.error("Unable to start peer server", exception)
+                );
                 return thread;
             });
-            peerServiceExecutor.execute(() -> peerServer.start(config.listenPort()));
+            peerServiceExecutor.execute(() -> peerServer.start(config.getBindAddress(), config.peerPort()));
```

Comment:
getPeerBindAddress and getPeerBindPort would look so better, why one with get why one with peer and one without it

Reply:
They have different names because bind address applies to other interfaces